framework/tlvf/yaml/tlvf/ieee_1905_1/eLinkMetricNeighborType.yaml

```diff
@@ -0,0 +1,10 @@
+#
+---
+_namespace: ieee1905_1
+
+eLinkMetricNeighborType:
+  _type: enum
+  _enum_storage: uint8_t
+  ALL_NEIGHBORS: 0x00
+  SPECIFIC_NEIGHBOR: 0x01
```

Comment:
If you declare eLinkMetricNeighborType in the same YAML file as the two TLV classes that use it then, as TLVF defines the eLinkMetricNeighborType **inside** one of the TLV C++ classes, you get a compilation error because it's undefined for the other TLV class.

Maybe there is some other workaround for this but I found moving eLinkMetricNeighborType to a different file the easiest.

Reply:
I didn't realize that. OK.